src/ClientData/include/ClientData/Constants.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CLIENT_DATA_CONSTANTS_H_
+#define CLIENT_DATA_CONSTANTS_H_
+
+#include <cstdint>
+namespace orbit_client_data {
+
+// The value returned by `AppInterface::ProvideScopeId` if the supplied
```

Comment:
I would rather state what the value is (rather than what it is not). We use this constant to identify scopes/timers where we don't have a corresponding Id (see IdProvider).